src/window/empire.c

```diff
@@ -328,12 +327,34 @@ static void draw_background(void)
     }
 }
 
+void window_empire_draw_trade_dots(const empire_object *trade_route, int x_draw_offset, int y_draw_offset)
```

Comment:
This whole thing mau not work  properly because it only draws straight lines. You may get a land route going over water instead of following the shore, for example.

Reply:
Yeah I think we would eventually want to allow users to draw their own trade route lines, but that would require some sort of UI (I wouldn’t make them code each little point in the XML). I think this gives us something that’s “good enough” for a first pass and then we can build something more flexible later.

Reply:
You can add waypoints to the xml though instead of having the user add every single point.

Reply:
Yeah that sounds good actually. Let’s go with that.